typeclasses/characters.py

```diff
@@ -84,6 +86,101 @@ def at_pre_unpuppet(self):
         if self.nattributes.has('combat_handler'):
             self.ndb.combat_handler.remove_character(self)
 
+    def return_appearance(self, looker):
+        """
+        This formats a description. It is the hook a 'look' command
+        should call.
+        Args:
+            looker (Object): Object doing the looking.
+        """
+        if not looker:
+            return ""
+
+        # get description, build string
+        name = self.get_display_name(looker)
+        per  = self.traits.PER
+
+        # These each do a skill check, but they always pass
+        # if you're looking at yourself
+        knows_race = skill_check(per) or looker == self
+        knows_archetype = skill_check(per) or looker == self
+        knows_health = skill_check(per) or looker == self
+        knows_stamina = skill_check(per) or looker == self
```

Comment:
What if you lowered the bar on the current health and required more perception/medicine points to see the character’s HP, and even more to see their total HP?

For example:

```
# 3
They seem a little roughed up.
# 5
They seem a little roughed up, and they’ve got 2 HP left.
# 7
They seem a little roughed up, and they’ve got 2/3 HP left.
```